fine_tuning_job.go

```diff
@@ -151,3 +157,61 @@ func (c *Client) ListFineTuningJobEvents(
 	err = c.sendRequest(req, &response)
 	return
 }
+
+type listFineTuningJobsParameters struct {
+	after *string
+	limit *int
+}
+
+type ListFineTuningJobsParameter func(*listFineTuningJobsParameters)
```

Comment:
Should we unify all our listing setters? (Maybe even with a small-ish API break?) 

@BrendanMartin Would you have the bandwidth to do this unification? (No worries if not, we can merge and update this afterward!)

@vvatanabe would love to hear your thoughts here.

The other listing setters: https://github.com/sashabaranov/go-openai/blob/master/fine_tuning_job.go#L100

Reply:
Unfortunately, I can't tackle this at the moment. To be honest, I don't even know what unification would look like.

Reply:
@BrendanMartin got it, thanks you!

Reply:
@sashabaranov Sorry for the late reply! I'd like to consolidate using a concise and generic structure like the one below. It's clear at a glance that it's for pagination purposes.

```
type PaginationQuery struct {
    After *string
    Limit *int
}
```

If we're going with this design, I can create a new PR after we merge this one.

Reply:
Merged into a non-master branch, let's take our time and unify the pagination approach